src/components/tools/text-tool.tsx

```diff
@@ -302,6 +302,12 @@ export default class TextToolComponent extends BaseComponent<IToolTileProps, ISt
     return this.props.model.content as TextContentModelType;
   }
 
+  private handleOnBlur = () => {
+    this.setState({ editing: false });
+    const change = {args:[{text: this.getContent().text}]};
+    Logger.logToolChange(LogEventName.TEXT_TOOL_CHANGE, 'update', change, this.props.model.id);
```

Comment:
I think we want to limit logging to situations in which the user has actually changed the text. I think if you captured the current text (`this.getContent().text`) in `handleFocus` you could then compare it in `handleBlur` to decide whether it had actually changed.

Reply:
Good idea! Updated.